src/java/map/FloatMask.java

```diff
@@ -572,6 +572,64 @@ public FloatMask removeAreasOfSpecifiedSizeWithLocalMaximums(int minSize, int ma
         return this;
     }
 
+    public FloatMask scaleFloatMaskBy(float scaleMultiplier) {
+        for (int x = 0; x < getSize(); x++) {
+            for (int y = 0; y < getSize(); y++) {
+                multiply(x, y, scaleMultiplier);
+            }
+        }
+        VisualDebugger.visualizeMask(this);
+        return this;
+    }
+
+    public FloatMask scaleFloatMaskTo(float newFloatMax) {
+        float scaleMultiplier = newFloatMax / getMax();
+        for (int x = 0; x < getSize(); x++) {
+            for (int y = 0; y < getSize(); y++) {
+                multiply(x, y, scaleMultiplier);
+            }
+        }
+        VisualDebugger.visualizeMask(this);
+        return this;
+    }
+
+    public FloatMask scaleFloatMaskFromTo(float oldFloatMax, float newFloatMax) {
+        float scaleMultiplier = newFloatMax / oldFloatMax;
+        for (int x = 0; x < getSize(); x++) {
```

Comment:
same use multiply(scaleMultiplier)

Also not clear on when you would use this over sclaeFloatMask

Reply:
or why you wouldnt just use multiply